README.md

```diff
@@ -54,6 +54,8 @@ The samples are built with Microsoft Compose libraries, [TwoPaneLayout](https://
 | | |
 | [NavigationRail](https://github.com/microsoft/surface-duo-compose-samples/tree/main/NavigationRail) | ![NavigationRail app icon](screenshots/navigation_rail.svg) | Nature-themed sample that shows how to use a combination of [NavigationRail](https://material.io/components/navigation-rail) and [BottomNavigation](https://material.io/components/bottom-navigation) when building apps for large screen and foldable devices in Jetpack Compose. |
 | | |
+| [VideoPlusChat](https://github.com/microsoft/surface-duo-compose-samples/tree/main/VideoPlusChat) |  | Video streaming platform sample that displays a video and accompanying chat. |
```

Comment:
Would you please also add an individual README for the sample with screenshots and description, like the other ones in this repo?